install.sh

```diff
@@ -114,20 +115,53 @@ EOF
                 echo "[i] File created."
 
                 # add cronjob
-                read -r -p "[?] Create a cronjob/systemd-service which updates /etc/hosts with new adservers once a week? [systemd/cronjob/N] " REPLY
+                read -r -p "[?] Create a cronjob/systemd-service which updates /etc/hosts with new adservers? [systemd/cronjob/N] " REPLY
                 case "${REPLY}" in
                     [Cc][Rr][Oo][Nn][Jj][Oo][Bb] | [Cr][Rr][Oo][Nn][Tt][Aa][Bb] | [Cc][Rr][Oo][Nn] | [Cc] ) # CRONJOB, CRONTAB, CRON, C
+                        read -r -p "[?] How often should the cronjob run? [weekly/DAILY/hourly/reboot] " FREQUENCY
+                        # set daily as default
+                        line="1 12 * * * root ${SCRIPT_DIR}/adaway-linux.sh"
```

Comment:
Can you set only the cron time expression in the switch case and add the `root ${SCRIPT_DIR}/adaway-linux.sh` only once for all cases in order to [not repeat yourself](https://en.wikipedia.org/wiki/Don't_repeat_yourself)?